openid-caep-specification-1_0.md

```diff
@@ -691,6 +706,56 @@ NOTE: The event type URI is wrapped, the backslash is the continuation character
 ~~~
 {: #device-compliance-change-examples-out-of-compliance title="Example: Device No Longer Compliant - Complex Subject + optional claims"}
 
+## Session Established {#session-established}
+Event Type URI:
+
+`https://schemas.openid.net/secevent/caep/event-type/session-established`
+
+The Session Established event signifies that the Transmitter has created a new session for a user for use with the Transmitter. Receivers may use this information for a number of reasons, including:
+
+* A service acting as a Transmitter can close the loop with the IdP after a user has been federated from the IdP
+* An IdP can detect unintended logins
+* A Receiver can establish an inventory of user sessions
+
+### Event Specific Claims {#session-established-event-specific-claims}
+The following optional claims MAY be included in the Session Established event:
+
+ip
+: The IP address of the user as observed by the Transmitter (**NOTE**, this can be different from the one observed by the Receiver for the same user because of network translation)
+
+fp_ua
+: Fingerprint of the user agent computed by the Transmitter. (**NOTE**, this is not to identify the session, but to present some qualities of the session)
+
+acr
```

Comment:
Even more important here I think because the values are organization specific (e.g. AAL2 is a NIST term)

Reply:
I've not been able to find the right URNs for this. I'm still researching this.

Reply:
yeah, same. I'm hoping @ve7jtb or @selfissued may be able to help

Reply:
@timcappalli I researched this for some work I am doing and you are spot on with it being org specific as this provides information about the strength of the authentication mechanism or process which pairs with the amr claim nicely. 

Something I was kicking around was urn:<company>:<identity>:loa<1|2|3> (e.g. urn:willywonka:identity:loa1) with a matching amr claim of pwd, as a full example)
I also was trying to incorporate [this](https://openid.net/specs/openid-provider-authentication-policy-extension-1_0.html#auth_policies) but it was a little too rigid.

Reply:
@iamseanodentity  BTW the correct handle on GitHub for me is @tulshi. 

Good point about the Assurance Level Change event. There, we are using a "namespace" field to distinguish between the different assurance level hierarchies. We don't have that here, and we could add some logic about it, or leave it open for now, since there is not registry that we can leverage.